packages/datadog-plugin-couchbase/test/index.spec.js

```diff
@@ -5,297 +5,304 @@ const semver = require('semver')
 const agent = require('../../dd-trace/test/plugins/agent')
 const proxyquire = require('proxyquire').noPreserveCache()
 
+const { VERSION } = process.env
+
 describe('Plugin', () => {
   let couchbase
 
   describe('couchbase', () => {
     let cluster
     let bucket
     let tracer
-    let collection
 
     before(() => {
       tracer = global.tracer = require('../../dd-trace')
     })
 
-    withVersions('couchbase', 'couchbase', '<3.0.0', version => {
-      let N1qlQuery
-      describe('without configuration', () => {
-        beforeEach(done => {
-          agent.load('couchbase').then(() => {
-            couchbase = proxyquire(`../../../versions/couchbase@${version}`, {}).get()
-            N1qlQuery = couchbase.N1qlQuery
-            cluster = new couchbase.Cluster('localhost:8091')
-            cluster.authenticate('Administrator', 'password')
-            cluster.enableCbas('localhost:8095')
-            bucket = cluster.openBucket('datadog-test', (err) => done(err))
+    if (VERSION === 'v2') {
```

Comment:
Why is a completely different test suite needed for v2 and v3? Could most of the code be reused and just have a different setup function for example?

Reply:
Hmm maybe, but there are just enough small differences in some tests, and some checks, that I decided to have them separate. I might just be able to remove the envvar check, but that would rely on keeping the change two comments above (restricting node versions, which for now is in the integration, but again, if there's a way I can move that to the tests, I definitely will!).